index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Microverse</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="./dist/css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <div class="wrapper">
+        <div class="header-inner">
+          <a href="/" class="header-logo">Welcome</a>
+          <a href="#" class="header-hamburger">
+            <img src="dist/images/hamburger.svg" alt="hamburger icon" />
+          </a>
```

Comment:
- [ ] For semantic reasons, it is always advisable to wrap your hamburger menu with the nav tag. Kindly wrap it with a nav.